web_teleop/frontend/src/web-teleop-app.html

```diff
@@ -0,0 +1,456 @@
+          user-select: none;
+      }
+    
+      h1, #camera, #btns {
+        margin-bottom: 1em;
+      }
+    
+      #forwardbtn {
+        margin-bottom: 0.6em;
+      }
+    
+      #interface {
+        margin: auto;
+        text-align: center;
+      }
+    
+      #btns {
+        display: flex;
+        align-items: flex-start;
+        flex-wrap: wrap;
+        height: 100%;
+      }
+    
+      #gripper, #torso {
+        width: 50%;
+      }
+    
+      #torso paper-slider, #gripper paper-slider {
+        margin: auto;
+      }
+    
+      #driving {
+        display: inline-block;
+        margin-bottom: 3em;
+      }
+
+      #camera {
+        width: 70%;
+        margin: auto;
+      }
+    
+      #camera img {
+          width: 100%;
+          margin: auto;
+          height: auto;
+      }
+    </style>
+
+    <ros-websocket auto ros="{{ros}}"
+      url="{{url}}"
+      on-connection="_handleConnection"
+      on-close="_handleClose"
+      on-error="_handleError"
+    ></ros-websocket>
+
+    <ros-topic auto
+      ros="{{ros}}"
+      topic="joint_state_republisher/torso_lift_joint"
+      msg-type="std_msgs/Float64"
+      last-message="{{torsoHeight}}"
+      on-message="_handleTorsoHeight"
+    ></ros-topic>
+
+    <ros-topic
+        id="baseTopic"
+        ros="{{ros}}"
+        topic="cmd_vel"
+        msg-type="geometry_msgs/Twist"
+    ></ros-topic>
+
+    <ros-service            
+        id="torsoService"
+        on-response="_handleTorsoSuccess"
+        on-fail="_handleTorsoError"
+        name="/web_teleop/set_torso"
+        ros="{{ros}}"
+        service-type="web_teleop/SetTorso"
+    ></ros-service>
+
+    <ros-service            
+        id="openGripperService"
+        on-response="_openGripperSuccess"
+        on-fail="_openGripperError"
+        name="/web_teleop/open_gripper"
+        ros="{{ros}}"
+        service-type="web_teleop/OpenGripper"
+    ></ros-service>
+
+    <ros-service            
+        id="closeGripperService"
+        on-response="_closeGripperSuccess"
+        on-fail="_closeGripperError"
+        name="/web_teleop/close_gripper"
+        ros="{{ros}}"
+        service-type="web_teleop/CloseGripper"
+    ></ros-service>
+
+    <ros-service            
+        id="setArmService"
+        on-response="_setArmSuccess"
+        on-fail="_setArmError"
+        name="/web_teleop/set_arm"
+        ros="{{ros}}"
+        service-type="web_teleop/SetArm"
+    ></ros-service>
+
+    <ros-service            
+        id="moveHeadService"
+        on-response="_moveHeadSuccess"
+        on-fail="_moveHeadError"
+        name="/web_teleop/move_head"
+        ros="{{ros}}"
+        service-type="web_teleop/MoveHead"
+    ></ros-service>
+
+    <div id="interface">
+      <h1>Fetch Teleop</h1>
+      <p style="color:#4B0082"><b>[[status]]</b></p>
+
+      <div id="camera">
+        <img src="//[[hostname]]:8080/stream?topic=/head_camera/rgb/image_raw"></img>
```

Comment:
RWS seems to use port 9999 as a default: https://github.com/hcrlab/rws/blob/92ef76bbaa9b96345c1ea916fab8601be65a2470/launch/rws.launch#L25

Reply:
Fetch is using `rws_fetch.launch` instead of [`rws.launch`](https://github.com/hcrlab/rws/blob/92ef76bbaa9b96345c1ea916fab8601be65a2470/launch/rws_fetch.launch#L18-L24), and it's using the default port.

Reply:
Also I'm using `web_video_server` here so the camera stream is by default on port 8080